ToDoApp/ToDoApp/CreateTaskView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  CreateTaskView.swift
+//  ToDoApp
+//
+//  Created by Trong Hieu Vuong on 2022-12-01.
+//
+
+import Foundation
+import SwiftUI
+
+struct CreateTaskView: View {
+    @StateObject private var createTaskViewModel = CreateTaskViewModel()
+   var body: some View {
+       VStack(spacing: 15) {
+           NavigationStack{
+               VStack(alignment: .leading, spacing: 15){
+                   Text("Task Title")
+                       .multilineTextAlignment(.leading)
+                   TextField("Insert Title", text: $createTaskViewModel.titleInput)
+                       .padding(.leading, 10.0)
+                     .textInputAutocapitalization(.words)
+                     .frame(width: 390.0, height: 40)
+                     .overlay(
+                        RoundedRectangle(cornerRadius: 14)
+                            .stroke(Color.accentColor, lineWidth: 2)
+                     )
+                   Spacer()
+                       .frame(height: 15)
+                   Text("Date of Completion")
+                   Button("Choose Date"){
+                       //Current stuck here, don't know how to click this button to open the date picker widget
+                   }
+                       .frame(width: 390.0, height: 40.0, alignment:Alignment.leading)
```

Comment:
again, no hardcoding width, alignment is also redundant
```suggestion
                       .frame(maxHeight: 40.0)
```

Reply:
This has been fixed.